### PR TITLE
Update salus_controls.js add PS600 pipe temerature sensor

### PR DESCRIPTION
**Pull request by johnjamesmarsh**
Added PS600 Pipe Temperature Sensor (temperature, battery_low & tamper). The vendor shows as Computime and the zigbeeModel is SS909ZB. I have it reporting the temperature, battery low and tamper variables correctly once per minute. It also supports OTA.

I have 2 devices working via zigb3ee2mqtt & HomeAssistant. Happy to help anyone else who wants to test it

### CODE REVIEW

File: devices/salus_controls.js [hunk 1]
```diff
@@ -86,6 +86,20 @@ module.exports = [
...
         ota: ota.salus,
     },
+    {
+        zigbeeModel: ['SS909ZB'],
+        model: 'PS600',
+        vendor: 'Salus Controls',
+        description: 'Pipe temperature sensor',
+        fromZigbee: [fz.temperature],
+        toZigbee: [],
+        exposes: [e.temperature(), e.battery_low(), e.tamper()],
```
Comment (Koenkk) [inline devices/salus_controls.js:96]: There are no fromZigbee converters which expose battery low and tamper so these should be removed or a converter should be added to fromzigbee

**Comment by johnjamesmarsh** [comment]
Removed battery low and tamper "expose" statements. They are there so will work on the correct config for them later and resubmit

**Comment by johnjamesmarsh** [comment]
Removed battery low and tamper "expose" statements. They are there so will work on the correct config for them later and resubmit

File: devices/salus_controls.js [hunk 2]
```diff
@@ -83,7 +83,21 @@ module.exports = [
         description: 'Door or window contact sensor',
         fromZigbee: [fz.ias_contact_alarm_1],
         toZigbee: [],
-        exposes: [e.contact(), e.battery_low(), e.tamper()],
+        exposes: [e.contact()],
```
Comment (Koenkk) [inline devices/salus_controls.js:86]: Why was `e.battery_low(), e.tamper()` removed here? `fz.ias_contact_alarm_1` supports these.

Comment (johnjamesmarsh) [reply devices/salus_controls.js:86]: That was a fat fingered fumble. I was changing the PS600 line and somehow modified the OS600 line too. Sorry about that, I'll resubmit.

**Comment by johnjamesmarsh** [comment]
I'm sorry, I'm making a real mess of this. I thought I knew what I was doing but the evidence does not support that. I'm just trying here to restore your version of the file

**Comment by johnjamesmarsh** [comment]
I think this version has everything correct in it. I have tested it back in my system and the sensor is still reporting temperature.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**